src/services/contracts/safeContracts.ts

```diff
@@ -132,7 +109,7 @@ export const getMultiSendCallOnlyContract = (
   const ethAdapter = createEthersAdapter(provider)
 
   return ethAdapter.getMultiSendCallOnlyContract({
-    singletonDeployment: getMultiSendCallOnlyContractDeployment(chainId),
+    singletonDeployment: getMultiSendCallOnlyDeployment({ network: chainId, version: safeVersion || undefined }),
```

Comment:
This is also an important fix, because it was getting 1.3.0 multisend by default which would break any time.